packages/vitaminjs-jest/src/index.js

```diff
@@ -0,0 +1,25 @@
+import { spawn } from 'npm-run';
+import jestConfig from './jestrc'
+
+export function registerCommand(program) {
+    return program
+        .command('jest [runnerArgs...]')
+        .description('Launch test suite with Jest')
+        .action((runnerArgs) => {
+            spawnJest({ runnerArgs: runnerArgs.join(' ') });
+        });
+};
+
+const spawnJest = ({ runnerArgs }) => {
```

Comment:
This function has potential for slippery escaping logic but is not tested, I think it should be if we want to be _ceinture bretelles_.